packages/types/App.d.ts

```diff
@@ -149,6 +149,8 @@ export type AppFrontendPayload = Omit<App, "key"> & {
     name?: string;
     installed?: boolean;
   }[];
+  /** Number of users who currently have this App installed */
```

Comment:
NIT: the variable explains what it is, no? 😆

Reply:
It's mostly for Intellisense

<img width="511" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/25ea2ee0-029e-482c-943c-44c39c07faad">